### PR TITLE
Added new features for CLI v3.0.4 and v3.0.5

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: modules/ROOT/pages/cli/anypoint-platform-cli.adoc [hunk 1]
```diff
@@ -4,6 +4,18 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cli, command line interface, command line, release notes, anypoint platform cli
 
+== Version 3.0.5 - January 25, 2019
+
+This release of Anypoint Platform CLI addresses the following issue:
+
+* Added support for pagination within Design Center.
```
Comment (noambenami) [inline modules/ROOT/pages/cli/anypoint-platform-cli.adoc]: Can a bit more detail be added here on which commands were modified and how?

Comment (fermujica) [reply modules/ROOT/pages/cli/anypoint-platform-cli.adoc]: See ec4f994
And let me know if that adds clarification.

File: modules/ROOT/pages/cli/anypoint-platform-cli.adoc [hunk 2]
```diff
@@ -4,6 +4,19 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cli, command line interface, command line, release notes, anypoint platform cli
 
+== Version 3.0.5 - January 25, 2019
+
+This release of Anypoint Platform CLI addresses the following issue:
```
Comment (noambenami) [inline modules/ROOT/pages/cli/anypoint-platform-cli.adoc]: Please don't add this everywhere. Not necessary.

File: modules/ROOT/pages/cli/anypoint-platform-cli.adoc [hunk 3]
```diff
@@ -4,6 +4,19 @@ include::_attributes.adoc[]
...
+
+* Added support for pagination within Design Center. +
+When running `designcenter project list`, you can now pass `--pageIndex` to set the number of pages to retrieve, and `--pageSize` to set the amount of results per page.
```
Comment (noambenami) [inline modules/ROOT/pages/cli/anypoint-platform-cli.adoc]: That doesn't makes sense. Should it not be "the exact page to retrieve"?

Comment (fermujica) [reply modules/ROOT/pages/cli/anypoint-platform-cli.adoc]: You are right. I misread the description in the documentation. Based on what the description in the product says, probably this is a more fitting description: b9b9ad8

File: modules/ROOT/pages/cli/anypoint-platform-cli.adoc (hunk 2 above)
Comment (fermujica) [reply modules/ROOT/pages/cli/anypoint-platform-cli.adoc]: I'm sorry, but we need to keep this for consistency with the release notes of all the products.

Comment (noambenami) [reply modules/ROOT/pages/cli/anypoint-platform-cli.adoc]: Oh sorry - I didn't notice that this wasn't in the cli repo.

File: modules/ROOT/pages/cli/anypoint-platform-cli.adoc [hunk 4]
```diff
@@ -4,6 +4,19 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: cli, command line interface, command line, release notes, anypoint platform cli
 
+== Version 3.0.5 - January 25, 2019
+
+This release of Anypoint Platform CLI addresses the following issue:
+
+* Added support for pagination within Design Center. +
+When running `designcenter project list`, you can now pass `--pageIndex` to set number of the page to retrieve from a paginated project, and `--pageSize` to set the amount of results per page.
```
Comment (noambenami) [inline modules/ROOT/pages/cli/anypoint-platform-cli.adoc]: "number" of results, not "amount" of results: "Amount" is for things that cannot be counted, like "amount of trash in the ocean".

**Pull request merged**